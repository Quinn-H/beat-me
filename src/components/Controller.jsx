import React from 'react'

const Controller = (props) => {
  const controller = props.controller
  const onClick = controller.onClick
  return(
  <td>
    <button type="button" className="btn btn-outline-danger" onClick={onClick}>
    </button>
  </td>
)
}

export default Controller
