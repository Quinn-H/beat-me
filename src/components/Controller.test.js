/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Controller from './Controller'

test('Controller renders correctly', () => {
  const stop = function () {}
  const start = function () {}
  const isPlaying = false
  const tree = renderer.create(
    <Controller start={start} stop={stop} isPlaying={isPlaying} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
