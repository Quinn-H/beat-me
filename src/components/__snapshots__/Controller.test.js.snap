exports[`test Controller renders correctly 1`] = `
<div>
  <button
    className="btn btn-outline-success"
    onClick={[Function]}>
    Start
  </button>
  <button
    className="btn btn-danger"
    onClick={[Function]}>
    Stop
  </button>
</div>
`;
